### ISSUE TITLE
.md or .rst

### ISSUE THREAD

**Issue by lheagy** [simpeg/community#4]
for the docs in this repo, it would be good to choose either `.md` or `.rst` and be consistent. 

My vote is for `.md` as these docs are quite simple and markdown is more widely used than `.rst`, and we can copy from GH issues, etc that have markdown styling. 

- vote with 👍  for `.md`
- vote with 👎 for `.rst`

**Comment by lheagy** [comment]
I am going to suggest we use `.md` to have things nicely rendered on github. if there are no objections by the end of the week, I will assume folks are okay with this :)

**Comment by prisae** [comment]
I say you phrased that survey very biased by giving one a :+1: and one a :-1:  ... ;-)

**Comment by lheagy** [comment]
there is only a small choice of emojis reactions here ;-)

**Comment by prisae** [comment]
I am fine with either. But I do think that with `.md` you sometimes reach the limits soon...

However, if it is without any sphinx then it might actually be more than enough.

**Comment by lheagy** [comment]
Yeah, I don't have strong opinions either way, but I think this repo should stay pretty simple in terms of what we need/want to include. The website-building should be confined to the individual repositories docs + the simpeg website itself

**Comment by prisae** [comment]
I just changed form :-1:  to :+1: - given your explanation that it is only for here, the community repo.

**Comment by lheagy** [comment]
Thanks @prisae! Since only you and I seem to have opinions on this, I will call that consensus :)
